Documentation/commands.md

```diff
@@ -129,6 +129,27 @@ $ journalctl -M rkt-bc3c1451-2e81-45c6-aeb0-807db44e31b4 -t redis
 
 Additionaly, logs can be programmatically accessed via the [sd-journal API](https://www.freedesktop.org/software/systemd/man/sd-journal.html).
 
+Currently there are known 2 main issues with loggin in rkt:
+* In some rare situations when an application inside the pod is writing to stdout/stderr instead of journal (i.e. nginx) there is no way to obtain logs.
+ The app should be modified so it will write to journal. In the case of nginx:
+ ```
+ error_log stderr;
+ 
+ http {
+     access_log syslog:server=unix:/dev/log main;
+     [...]
+ }
+ ```
+ should be added to ```/etc/nginx/nginx.con```
+
+* With the newest versions of etcd and rkt, etcd logs can be accesed only via
```

Comment:
I think this point should be made more generic first, and etcd just an example of it.

The real problem to be described is that services can write directly to journald (in the pod), resulting in their log entries being scoped by machine UUID.